share/man/man7/hier.7

```diff
@@ -157,20 +165,31 @@ infinite loop that accepts anything and contains nothing
 .It Pa nvd0
 first NVMe storage device using NVMe namespaces
 .It Pa pts/
-pseduo-terminals
+pseduo-terminals; see
+.Xr pts 4
 .It Pa random
 source of weak randomness; see
 .Xr random 4
 .It Pa sa0
 first tape drive
 .It Pa usb/
 USB busses
+.It Pa vmm/
+active
+.Xr bhyve 8
+virtual machines
+.It Pa zvol/
+.Xr zfs 8
+volumes
 .El
 .It Pa /entropy
 provides initial state to RNG; see
-.Xr save-entropy 8
+.Va entropy
+in
+.Xr rc.conf 5
```

Comment:
I think the most appropriate place to mention it is in the entropy documentation in `rc.conf(5)`. Please excuse the noise and churn, that I didn't think of that the first time.

Reply:
I think we should retain the xref to safe-entropy since it more directly explains things. It likely needs to include all the variables, though rc.conf would then duplicate things... which is a bit of a mess right now in terms of where one looks for the canonical list (mostly rc.conf.5, but sometimes there's other places where they are documented)